generate-stackbrew-pr.sh

```diff
@@ -108,16 +117,25 @@ if updated; then
 	info "Pushing..."
 	git push "https://$GITHUB_API_TOKEN:x-oauth-basic@github.com/$ORIGIN_SLUG.git" -f "$BRANCH_NAME" 2> /dev/null || fatal "Error pushing the updated stackbrew"
 
+	cd - && rm -rf $gitpath
+
 	info "Creating Pull request"
 	pr_response_payload="$(curl -H "$(auth_header)" \
 		-s \
 		-X POST \
 		-d "$(pr_payload)" \
 		"https://api.github.com/repos/$UPSTREAM_SLUG/pulls")"
 
-	url="$(echo "$pr_response_payload" | jq .html_url)"
+	url="$(echo "$pr_response_payload" | jq -r .html_url)"
 	if [ "$url" != "null" ]; then
 		info "Pull request created at $url"
+
+		info "Creating Commit Comment"
+		curl -H "$(auth_header)" \
+		-s \
+			-X POST \
+			-d "$(comment_payload "$url")" \
+			"https://api.github.com/repos/$DOCKER_SLUG/commits/$COMMIT_ID/comments" > /dev/null
```

Comment:
I suppose we don't need to redirect standard output to `/dev/null` as `-s` is passed to `curl`?

Reply:
Yeah, `-s` suppresses the progress update but not the output itself.

Reply:
hmmm, I think we need to output to know the result?

Reply:
This one I don't think so although i could output the error message if present.